src/controllers/articleRatingController.js

```diff
@@ -0,0 +1,75 @@
+import models from '../db/models';
+
+const { Rating } = models;
+
+
+/**
+ * @description class responsible for rating artcles
+ *
+ * @class ArticleRatingController
+ */
+class ArticleRatingController {
+  /**
+   * @description - This method is responsible for rating user article
+   *
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   *
+   * @returns {object} - object representing response message
+   *
+   * @memberof ArticleRatingController
+   */
+  static async rateArticle(request, response) {
+    const { rating } = request.body;
+    const articleSlug = request.params.slug;
+
+    try {
+      const result = await Rating.create({
+        articleSlug,
+        rating
+      });
+      response.status(201)
+        .json({
+          message: 'Rating recorded successfully',
+          result
+        });
+    } catch (error) {
```

Comment:
Hi Marcus, there is a helper function that returns error messages as stated in the README in the helpers' folder [here](https://github.com/andela/balder-ah-backend/blob/staging/src/helpers/index.js). I would love you look into it.

Reply:
Cool. I will look into it. Thanks for the heads up.